content/chapters/compute/lecture/demo/create-process/multiple_forks.c

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <fcntl.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/types.h>
+#include <sys/wait.h>
+#include <unistd.h>
+
+#include "utils.h"
+
+int main(void)
+{
+	pid_t pid;
+	int rc;
+	int exit_code;
+
+	fork();
```

Comment:
This doesn't help show the process hierarchy which is what you want to have with this demo.

I suggest you show the PID of the current process, before / after each `fork()`. And also, after all forks, do a `wait_for_input()` and then do a `pstree -p` or `ps -H` to show the process hierarchy.